composer.json

```diff
@@ -20,6 +20,7 @@
 			"inc/environment_indicator/namespace.php",
 			"inc/cloudfront_media_purge/namespace.php",
 			"inc/fluent_bit/namespace.php",
+			"inc/session_handler/class-disallowed-session-handler.php",
```

Comment:
I never noticed that we had `inc/class-loggerexceptionhandler.php` below. Strictly speaking that should be in the `classmap` section above. It could be converted to the following and the class files could be removed from this files section:

```
"classmap": [
	"inc/"
],
```

We can deal with that in a follow up I think as it might make backporting difficult.